Makefile.include

```diff
@@ -485,6 +488,7 @@ DIRS += $(EXTERNAL_MODULE_DIRS)
 
 # Define dependencies required for building (headers, downloading source files,)
 BUILDDEPS += $(RIOTBUILD_CONFIG_HEADER_C)
+BUILDDEPS += $(KCONFIG_GENERATED_AUTOCONF_HEADER_C)
```

Comment:
to increase modularity I would suggest to move your changes to `BUILDDEPS` and `CFLAGS` to `makefiles/kconfig.mk`

Reply:
Changes are now in `makefiles/kconfig.mk`.